src/Momento.Sdk/ISimpleCacheClient.cs

```diff
@@ -195,6 +195,9 @@ public interface ISimpleCacheClient : IDisposable
     /// <inheritdoc cref="SetAsync(string, byte[], byte[], TimeSpan?)"/>
     public Task<CacheSetResponse> SetAsync(string cacheName, string key, byte[] value, TimeSpan? ttl = null);
 
+    /// <inheritdoc cref="SetAsync(string, byte[], byte[], TimeSpan?)"/>
+    public Task<CacheSetResponse> SetAsync(string cacheName, byte[] key, string value, TimeSpan? ttl = null);
```

Comment:
nice find!

Can I recommend two small cleanups while you are in here?

1. Can you re-organize these so that all the Sets are together, all the Gets are together, etc.?  It seems confusing and error prone for us to have spread them out this way (not your fault!)  If that seems like a ton of work it's okay to punt it for now.
2. Can you see what happens if you change this line:
https://github.com/momentohq/client-sdk-dotnet/blob/main/tests/Integration/Momento.Sdk.Tests/Fixtures.cs#L15

to use the Interface instead of the class?  I think that would catch if there are any other places where we've missed stuff in the interface?

Reply:
Thank you! 
1. Fixed the re-organization of sets/gets/deletes
2. Noob alert: If we use the interface, not sure how we can set the Logger - https://github.com/momentohq/client-sdk-dotnet/blob/main/tests/Integration/Momento.Sdk.Tests/Fixtures.cs#L26

Reply:
Awesome, thanks!

Re: 2. -> you will need to change that constructor call from `new(...)` to `new CacheClient(...)`.  The `new(...)` syntax is some fancy C# syntax sugar that allows you to omit the class name if the compiler can infer it.  You are correct that when you change the return type to the interface, that fancy syntax sugar will no longer work because the compiler can no longer infer the type that you are trying to construct.  But you can just explicitly specify it rather than using the syntax sugar.  Good question!